pages/dkp/kommander/1.3/release-notes/1.3.3/index.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Release Notes Kommander 1.3.3
+title: Release Notes Kommander 1.3.3
+menuWeight: 40
+excerpt: View release-specific information for Kommander 1.3.3
+enterprise: false
+---
+
+**D2iQ&reg; Kommander&reg; version 1.3.3 was released on x, May 2021.**
+
+[button color="purple" href="https://support.d2iq.com/s/entitlement-based-product-downloads"]Download Konvoy[/button]
+
+To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [install](/dkp/konvoy/latest/install/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+# Release Summary
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+# Known issues
```

Comment:
We don't need this section, because this IS the upgrade path/fixed version.

That being said, it might just be nice to have this line edited, because people might just land on this page not knowing for sure what to do with their Kubecost business.

So, I don't know? Whatever you all think is best @gracedo and @cmastr @russurquhart1 @JohnB64-JBV 
We could probably wipe this section for 1.3.3 or just apply the change below. Or maybe not - after re-reading it, it is pretty clear. Either way.

Reply:
yeah I think we should add SOMETHING to release notes. We could add it as a fix, or we can add it as a known issue with a workaround.